source/agora/consensus/data/Transaction.d

```diff
@@ -53,6 +55,9 @@ public struct Transaction
     /// The list of newly created outputs to put in the UTXO
     public Output[] outputs;
 
+    /// The data to store
+    public ubyte[] data;
```

Comment:
I was able to confirm that the deserializer works well even if I use ubyte[] without having to implement fromBinary by making a separate type.
Moreover, when printing as JSON characters, I finally implemented the code using ubyte[] because toString, fromString, is not necessary if JSON array is used without hexstring.